### PR TITLE
Updated NGS tutorial for GTN Smörgåsbord

### PR DESCRIPTION
**Pull request by nekrut**
Completely redone NGS tutorial based on @tnabtaf @blankenberg and @mvdbeek SARS-CoV-2 material

### CODE REVIEW

**Comment by nekrut** [comment]
This is almost done. Just waiting on Jupyter section. Will remove `work-in-progress` once done.

**Comment by nekrut** [comment]
Don't understand why lint is failing. The images are there and I do not have images without alt text

**Comment by hexylena** [comment]
I see some alt text changes in the latest commit. I'm guessing it was that, hopefully it passes how :)

**Comment by hexylena** [comment]
It seems the failing images were in the introduction topic. I'm guessing you copied and pasted text from the introduction into this variant analysis topic tutorial? I've updated the links, hopefully they work now :)

**Comment by nekrut** [comment]
> It seems the failing images were in the introduction topic. I'm guessing you copied and pasted text from the introduction into this variant analysis topic tutorial? I've updated the links, hopefully they work now :)

Ahh . .. of course. Thanks

**Comment by hexylena** [comment]
Oh, my paths are wrong still. Ok, I can fix this tomorrow, and we'll get this merged.

**Comment by hexylena** [comment]
Thanks for the update @nekrut, this is great to see :)

File: topics/variant-analysis/tutorials/sars-cov-2/tutorial.md [hunk 1]
```diff
@@ -67,14 +93,14 @@ SRA can be reached either directly through it's website, or through the tool pan
 
 > ### {% icon comment %} Comment
 >
-> Initially the tool panel option exists only on the [usegalaxy.org server](https://usegalaxy.org/).  Support for the direct connection to SRA will be included in the 20.05 release of Galaxy
+> Initially the tool panel option for accessing SRA exists only on the [usegalaxy.org server](https://usegalaxy.org/).  Support for the direct connection to SRA will be included in the 20.05 release of Galaxy
```
Comment (nsoranzo) [inline topics/variant-analysis/tutorials/sars-cov-2/tutorial.md:96]: This comment seems outdated, can be removed?

File: topics/variant-analysis/tutorials/sars-cov-2/tutorial.md [hunk 2]
```diff
@@ -67,14 +93,14 @@ SRA can be reached either directly through it's website, or through the tool pan
...
 {: .comment}
 
 
 > ### {% icon hands_on %} Hands-on: Explore SRA Entrez
 >
-> 1. Go to [usegalaxy.org](https://usegalaxy.org/)
-> 1. If your history is not already empty, than start a new history (see here for more on Galaxy histories)
+> 1. Go to your Galaxy instance of choice such as one of the [usegalaxy.org](https://usegalaxy.org/), [usegalaxy.eu](https://usegalaxy.eu), [usegalaxy.org.au](https://usegalaxy.org.au) or any other. (This tutorial uses usegalaxy.org).
+> 1. If your history is not already empty, than start a new history (see [here](https://training.galaxyproject.org/training-material/topics/galaxy-interface/tutorials/history/tutorial.html) for more on Galaxy histories)
```
Comment (nsoranzo) [inline topics/variant-analysis/tutorials/sars-cov-2/tutorial.md:103]: I'd use the `create_new_history` snippet here instead.

File: topics/variant-analysis/tutorials/sars-cov-2/tutorial.md [hunk 3]
```diff
@@ -201,20 +227,90 @@ The `SRA` dataset is not sequence data, but rather *metadata* that we will use t
...
+>    > Galaxy may have an overwhelming amount of tools installed. To find a specific tool type the tool name in the tool panel search box to find the tool.
 >    {: .tip}
+> 1. Make sure the `SraRunInfo.csv` dataset we just uploaded is listed in the {% icon param-file %} "*Select lines from*" field of the tool form.
+> 1. In "*the pattern*" field enter the following expression &rarr; `SRR12733957|SRR11954102`. These are two accession we want to find separated by the pipe symbol `|`. The `|` means `or`: find lines containing `SRR12733957` **or** `SRR11954102`.
+> 1. Click `Execute` button.
+> 1. This will generate a file containing two lines (well ... one line is also used as the header, so it will appear the the file has three lines. It is OK.)
+> 1. Cut the first column from the file using {% icon tool %} "**Cut**" tool, which you will find in **Text Manipulation** section of the tool pane.
+> 1. Make sure the dataset produced by the previous step is selected in "*File to cut*" field of the tool form.
```
Comment (nsoranzo) [inline topics/variant-analysis/tutorials/sars-cov-2/tutorial.md:292]:
```suggestion
> 1. Make sure the dataset produced by the previous step is selected in the "*File to cut*" field of the tool form.
```

File: topics/variant-analysis/tutorials/sars-cov-2/tutorial.md (hunk 2 above)
Comment (nekrut) [reply topics/variant-analysis/tutorials/sars-cov-2/tutorial.md:103]: Hmmm ... my tutorial is in `introductions`. I did not mean to touch the `topics/variant-analysis/tutorials/sars-cov-2/tutorial.md` version

File: topics/variant-analysis/tutorials/sars-cov-2/tutorial.md (hunk 3 above)
Comment (nekrut) [reply topics/variant-analysis/tutorials/sars-cov-2/tutorial.md:292]: same here

Comment (nekrut) [reply topics/variant-analysis/tutorials/sars-cov-2/tutorial.md:292]: same here

File: topics/variant-analysis/tutorials/sars-cov-2/tutorial.md (hunk 1 above)
Comment (nekrut) [reply topics/variant-analysis/tutorials/sars-cov-2/tutorial.md:96]: I'd leave this for now as we will be modifying this again once SRA fixes Run Selector

File: topics/variant-analysis/tutorials/sars-cov-2/tutorial.md (hunk 3 above)
Comment (nekrut) [reply topics/variant-analysis/tutorials/sars-cov-2/tutorial.md]: fixed

File: topics/variant-analysis/tutorials/sars-cov-2/tutorial.md (hunk 2 above)
Comment (nekrut) [reply topics/variant-analysis/tutorials/sars-cov-2/tutorial.md:103]: This portion is commented out for now

**Comment by hexylena** [comment]
Just ping me when it's done @nekrut and I'll have a look :)

**Comment by nekrut** [comment]
done

Anton Nekrutenko
Professor of Biochemistry
and Molecular Biology
Penn State University
*http://nekrut.bx.psu.edu <http://nekrut.bx.psu.edu>*
(814) 826-9628

**Review by hexylena — APPROVED** [review]
Super!

**Pull request merged**